packages/explat-client/src/create-explat-client.ts

```diff
@@ -35,6 +35,16 @@ export interface ExPlatClient {
 	 *
 	 */
 	dangerouslyGetExperimentAssignment: ( experimentName: string ) => ExperimentAssignment;
+
+	/**
+	 * INTERNAL USE ONLY
+	 */
+	internalUseOnlyLogError: (error: Record< string, string > & { message: string; properties?: Record< string, unknown > }) => void;
+
+	/**
+	 * INTERNAL USE ONLY
+	 */
+	internalUseOnlyIsDevelopmentMode: boolean;
```

Comment:
How about we expose the full config as an attribute instead? It'd be less verbose and allow us to access the same functionality.

Reply:
Yeah thats a good idea! I will go with just `config` for now.